examples/java/src/main/java/org/apache/beam/examples/complete/game/StatefulTeamScore.java

```diff
@@ -115,9 +114,14 @@ private static Map<String, FieldInfo<KV<String, Integer>>> configureCompleteWind
   public static void main(String[] args) throws Exception {
 
     Options options = PipelineOptionsFactory.fromArgs(args).withValidation().as(Options.class);
+
     // Enforce that this pipeline is always run in streaming mode.
     options.setStreaming(true);
-    ExampleUtils exampleUtils = new ExampleUtils(options);
+    runStatefulTeamScore(options);
```

Comment:
Thinking about this at a high level, we can test it like this: instead of a static function `runTeamScore` that creates the pipeline and runs it, you should instead move the logic into a PTransform `TeamScore` and the logic is in the `expand()` method. Then ITs and main() can run it in different ways. This will test the core logic. It will not test the command line interface.